src/features/bookings/components/BookingDetailsTicketContent.tsx

```diff
@@ -107,12 +160,19 @@ const Title = styled(Typo.Title3).attrs(getHeadingAttrs(1))({
   maxWidth: '100%',
 })
 
-const TicketInnerContent = styled.View(({ theme }) => ({
-  justifyContent: 'center',
+const TicketInnerContent = styled.View({ 
+  paddingTop: getSpacing(6),
   paddingHorizontal: getSpacing(5),
-  minHeight: theme.ticket.minHeight,
   width: '100%',
-}))
+ })
+
+const TicketInfo = styled(Typo.Body).attrs(getHeadingAttrs(1))({
+  textAlign: 'center',
+  maxWidth: '100%',
+  paddingBottom: getSpacing(6),
+})
+
+const TicketCollectDelay = styled(Typo.Body).attrs(getHeadingAttrs(1))(({ theme }) => ({ fontFamily: theme.fontFamily.bold }))
```

Comment:
On a déjà un titre `h1` avant et cette info ne mérite pas vraiment d'être un titre donc tu peux enlever `.attrs(...)` 
```suggestion
const TicketCollectDelay = styled(Typo.Body)(({ theme }) => ({ fontFamily: theme.fontFamily.bold }))
```